lib/src/gutter/gutter.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+
+import '../code/code.dart';
+import '../line_numbers/line_number_style.dart';
+
+const _foldingColumnWidth = 16.0;
+
+class GutterWidget extends StatelessWidget {
+  const GutterWidget({
+    required this.code,
+    required this.style,
+  });
+
+  final Code code;
+  final LineNumberStyle style;
+
+  @override
+  Widget build(BuildContext context) {
+    final tableRows = [
+      for (int i = 1; i <= code.lines.length; i++)
+        TableRow(
+          children: [
+            Text(
+              '$i',
+              style: style.textStyle,
+              textAlign: style.textAlign,
+            ),
+            const SizedBox(),
```

Comment:
What is this SizedBox for?

Reply:
To reserve space. Its width comes from `_foldingColumnWidth`.
Without it, the entire width is taken by line numbers.